sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/aio/_session_async.py

```diff
@@ -90,10 +88,17 @@ async def _set_state(self, new_state):
             return
         previous_state = self.state
         self.state = new_state
-        _LOGGER.info("Session state changed: %r -> %r", previous_state, new_state, extra=self.network_trace_params)
+        _LOGGER.info(
+            "Session state changed: %r -> %r",
+            previous_state,
+            new_state,
+            extra=self.network_trace_params,
+        )
         futures = []
         for link in self.links.values():
-            futures.append(asyncio.ensure_future(link._on_session_state_change())) # pylint: disable=protected-access
+            futures.append(
```

Comment:
same as below, we can make this in to list comprehension :)

Reply:
will probably hold off on this until a separate PR where we fix list comprehension/fstrings/etc. ,like you mentioned.

Reply:
agree